### PR TITLE
Fix track speed not being reset after exiting the editor

### LINKED ISSUE #7468: Track speed modification in editor is applied globally
Connecting it to the epic: #6686

When we slow down the song in the editor to 50% or 75% and we exit the editor, we are able to play a map with that speed: https://streamable.com/sncb9

Source: https://osu.ppy.sh/comments/1004130

### PR DESCRIPTION
**Pull request by Wieku**
Fixes #7468

### CODE REVIEW

**Comment by smoogipoo** [comment]
This isn't the correct direction for this. Since `WorkingBeatmap` has now been moved to `Editor`, we can add a `GameplayClockContainer` to the editor and have the tabs bound to `UserPlaybackRate`.

There's a lot that this PR isn't doing, such as not removing the bindable on exit, for which `Editor` would have to call `GameplayClockContainer.StopUsingBeatmapClock()` as per what's done in `Player`.

Basically just look towards `Player` for implementation references.

**Comment by Wieku** [comment]
Never thought about this, I will look into this later
